File: 2020-11/abc149_c.py
import math

def is_prime(n):
    if n == 1: return False

    for k in range(2, int(math.sqrt(n)) + 1):
        if n % k == 0:
            return False

    return True

n = int(input())

while True:
    if is_prime(n):
        print(n)
        exit()
    else:
        n += 1

File: 2021-02/abc149_c.rs
use whiteread::parse_line;
use std::process;

fn isPrime(num: usize) -> bool {
    if num == 1 {
        return  false;
    }

    for i in 2..((num as f32).sqrt() as usize + 1) {
        if num % i == 0 {
            return false;
        }
    }
    return true;
}

fn main() {
    let n: usize = parse_line().unwrap();

    for i in n..100004 {
        if isPrime(i) {
            println!("{}", i);
            process::exit(0x0100);
        }
    }
}
